parlacards/serializers/organization.py

```diff
@@ -12,7 +12,9 @@
 
 class OrganizationBasicInfoSerializer(CommonCachableSerializer):
     def calculate_cache_key(self, instance):
-        return f'OrganizationBasicInfoSerializer_{instance.id}_{instance.updated_at.strftime("%Y-%m-%dT%H:%M:%S")}'
+        last_membership_updated_at = instance.personmemberships_children.latest("updated_at").updated_at
```

Comment:
kaj se zgodi če ne obstaja nobem membership tu? a je to sploh možno?

Reply:
dodal try except da ne crkne....